Sources/Proxy/Core/View2DProxy/index.js

```diff
@@ -124,7 +124,9 @@ function vtkView2DProxy(publicAPI, model) {
     if (rep.setSlicingMode) {
       rep.setSlicingMode('XYZ'[model.axis]);
     }
-    publicAPI.bindRepresentationToManipulator(rep);
+    if (rep.getProxyId) {
```

Comment:
Might be better to add this check inside the `bindRepresentationToManipulator()` call, so that it is handled in other places where it is called, such as line 140.